src/routes/kv.py

```
@@ -0,0 +1,69 @@
+  return key
+
+
+class KvUpdateBody(BaseModel):
+  value: str
+
+
+@kv_router.get("/{key}")
+async def get_kv_value(
+  key: str,
+  session: SessionDep,
+  request: Request,
```

Ruff (0.13.3)

Unused function argument: `request`

(ARG001)